src/utils.ts

```diff
@@ -0,0 +1,125 @@
+import { publish as _publish } from 'libnpmpublish'
+import got from 'got'
+import ssri from 'ssri'
```

Comment:
Codacy has a fix for the issue: [Replace `'ssri'` with `"ssri";`](https://app.codacy.com/gh/joebowbeer/regsync/pullRequest?prid=9263162)
```suggestion
import ssri from "ssri";
```